owncloudApp/src/main/java/com/owncloud/android/ui/asynctasks/CopyAndUploadContentUrisTask.java

```diff
@@ -257,10 +256,7 @@ protected void onPostExecute(ResultCode result) {
                     default:
                         messageId = R.string.common_error_unknown;
                 }
-                String message = String.format(
-                        mAppContext.getString(messageId),
-                        mAppContext.getString(R.string.app_name)
-                );
+                String message = mAppContext.getString(messageId);
```

Comment:
If we fix the lint error this way, we will need to update every translation of this key: `uploader_error_message_read_permission_not_granted` and it is translated to several languages.

We could,  for example, retrieve the string message inside the switch, so that, we won't need to update every single translation. What do you think @hannesa2 ?

Reply:
The root cause is to use several translated strings, where one has a placeholder for "Owncloud".
What's the point to have the (dynamic ?) app name there ?

To be honest, I would solve this root cause (this pull request) and change translated string in that way to remove this placeholder.
This would be a nice, but not a mandatory step.

Anyway, you can transform this pull request in any way you want (or later in an own pull request). The branch is in origin repository

Reply:
I know which is the root cause. I agree that it may be unnecessary to have that placeholder, but this PR does not fix the issue completely. Some translations will expect an argument for that key and we are not providing it.

Thanks anyway to report the lint error 👍